### PR TITLE
Update to work with wagtail 2.9+

### PR DESCRIPTION
**Pull request by tombola**
The requests middleware that adds a site property to request has been removed.

https://docs.wagtail.io/en/stable/releases/2.9.html#upgrade-considerations

### CODE REVIEW

**Comment by seb-b** [comment]
Thanks for this. I'm not sure what's happened to travis but the test results aren't showing up in the pull request. There's just a couple of isort things that need clearing up:
https://travis-ci.org/github/neon-jungle/wagtail-schema.org/jobs/717891424

File: wagtailschemaorg/jinja2tags.py [hunk 1]
```diff
@@ -1,20 +1,20 @@
 import jinja2
 from jinja2.ext import Extension
-
```
Comment (seb-b) [inline wagtailschemaorg/jinja2tags.py:3]: Need this blank line back in fro isort

File: wagtailschemaorg/templatetags/wagtailschemaorg_tags.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from django.template import Library
-
```
Comment (seb-b) [inline wagtailschemaorg/templatetags/wagtailschemaorg_tags.py:2]: same as above, needed for isort

**Comment by tombola** [comment]
> the test results aren't showing up in the pull request

Assume you are not hitting the github rate limit :)

**Comment by seb-b** [comment]
Released in 2.1.0 :tada:

**Pull request merged**